### ISSUE TITLE
Refactor ProblemList_openjdk**-openj9.txt to openj9 | ibm folder.

### ISSUE THREAD

**Issue by sophia-guo** [adoptium/aqa-tests#5835]
Vendor specific ProblemLists (excludes/vendors/vendor/ProblemList_openjdk*.txt) is used for handling vendor specific features . Common temporary problemlists stays in openjdk/excludes/ProblemList_openjdk*.txt. 

Currently openj9|IBM problemlists is an exception, which has a different name for problemlist. Most of the exclusions in problemlist is duplicated in openj9 problemlist. And when re-enable both files need to be updated. 

Suggest to refactor those files. Move openj9 problemlist to openj9|ibm as a vendor folder. Renaming the "-openj9" problemlist as normal ones. Removing all duplicate exclusions in openj9 problemlist.

This can eliminate the redundancy, ease the maintenance, improve the consistency and make openj9 problemlist specific, clear to its own issues.

**Comment by sophia-guo** [comment]
openj9 sdk has following properties
 "-DJDK_IMPL=openj9" "-DJDK_VENDOR=eclipse"
  "-DJDK_IMPL=openj9" "-DJDK_VENDOR=ibm"

**Comment by llxia** [comment]
Thanks for trying to optimize the workflow. It would be great if we could reduce the common work and reduce the duplication.

Currently, the exclusion workflow is the following:
- default ProblemList (in openjdk repo)
- ProblemList in aqa repo  https://github.com/adoptium/aqa-tests/tree/master/openjdk/excludes <== **we have ProblemList for hotspot (ProblemList_openjdkXXX.txt) and ProblemList for openj9 (ProblemList_openjdkXXX-openj9.txt)** 
- feature ProblemList based on TEST_FLAG (in openjdk repo)
- vendor ProblemList in aqa repo https://github.com/adoptium/aqa-tests/tree/master/openjdk/excludes/vendors
https://github.com/adoptium/aqa-tests/blob/c42bb0304f895bd393767d7dd687d35a85ad3857/openjdk/playlist.xml#L235-L238

When running OpenJDK tests against openj9/ibm, we do not use ProblemList for hotspot (ProblemList_openjdkXXX.txt) at all. 

This issue suggests moving Openj9 ProblemList (ProblemList_openjdkXXX-openj9.txt) in aqa repo to vendor dir and using ProblemList for hotspot (ProblemList_openjdkXXX.txt) as the common one. While this could reduce the work for common excludes (i.e., issues with OpenJDK https://bugs.openjdk.org/), this also means any exclude in ProblemList_openjdkXXX.txt that is not common will affect openj9 runs (i.e, Adoptium infra specific excludes or Terium specific excludes, etc), which may not be ideal.

In summary, I am all for simplifying and optimizing the workflow, but IMHO we need to have a real common ProblemList (i.e., common issues with OpenJDK only https://bugs.openjdk.org/). And we could also have a common ProblemList for common infra issues if needed. Any other infra or JVM-specific excludes should go to vendor excludes. (If we want to be sophisticated, we may want to have an infra ProblemList per site ).

cc @smlambert @pshipton @JasonFengJ9 for any comments and suggestions.

**Comment by sophia-guo** [comment]
This can also be done for newer version (25+) first and then for older versions if desired.
